assets/model_monitoring/components/generation_safety_quality/annotation_compute_metrics/spec.yaml

```diff
@@ -60,23 +60,20 @@ conf:
   spark.synapse.library.python.env: |
     channels:
       - defaults
-      - conda-forge
     dependencies:
       - python=3.10
       - pip:
-          - azure-cli-core~=2.45.0
```

Comment:
this is old env before my merged PR - it looks like this PR needs to be synced with latest main to see the actual changes here?